docs/contribute/workflows/signup.md

```diff
@@ -2,28 +2,36 @@
 
 ```mermaid
 sequenceDiagram
+    %% https://mermaid.live/edit#pako:eNqNVE2P2jAQ_SsjXzjsUu6RulJ3iSpOWxHSU6TKaw_BSrBTf1BFiP_esRNoYFltOYXM87x5b158ZMJIZBlz-DugFrhUvLZ8X2mgHw_e6LB_Qzv877j1SqiOaw9lDtxB6dDme67aO4DiDLhTW71_t95E_Dpor_b4vrx8Ht6VxfzpqVxlkGuP1oFET_QOjAanaj0PHR2r8Qb80irROCgGxFhcUfFhvcngx2uxgQUXwhD7wmKtnD-Pvd4QavlMLSxyjxNBqRKPf0eNNtaSE_ATrdoqwb2imTamQX1Rc5YH3txad-lY5hkUqCVg6iaMJoFa6RoO08at0s2_mVOpTycWRx85T-MKqdGUdHXFRnTzaM8RnOc-uAxmpuH9DMbTNEalJ25FbLEzf2D2skPRQG-CHeacTV3PY-OyyEZDLApUB5STnTzETq8davcfGgZqardUrmt57xIXpaWJrgTyMUoTccVQsWR_P1BXDN6CpxDfi8Mn2A_SMUx5PeAlCckVB2MRlIOtoeF2CK2pa5SgdJr3JloxCl-S8F_DjlF-9TaMdrbGdKkPcrED8kxG2UoflMeb7MRm36S8eOLx_ClfQZbYIsX1wxZlJ2OceZDKx8knYZgGhzayRqlovR5mi1ml2SPboyUdkm6UYwRXjMTT58wyepTcNhWr9IlwIVHkRGAsy7a8dfjI4n1T9FqwLKo_g8YraUSd_gKYcISX
     autonumber
-    participant UserEmail
-    participant User
+    participant UE as UserEmail
+    participant US as User
     participant UI
-    participant Runtime
+    participant RT as Runtime
     participant DB
-    User->>UI: Enters details on sign-up page
-    User->>UI: Clicks Sign-up
-    UI->>+Runtime: POST /account/register
-    Runtime->>DB: Create User
-    Runtime->>Runtime: Generate Email Verification Token
-    Runtime->>UserEmail: Send email containing verification link /account/verify/{token}
-    Runtime-->>-UI: { status: 'okay' }
+    US->>UI: Enters details on sign-up page
+    US->>UI: Clicks Sign-up
+    UI->>+RT: POST /account/register
+    RT->>DB: Create User
+    RT->>RT: Generate Email Verification Token
+    par Runtime to UserEmail
+        RT->>UE: Send email containing verification link /account/verifyemail/{token}
```

Comment:
Is there a reason for having changed this initial url? I would personally have chosen to keep it as `/account/verify/{token}` rather than a separate end point.

Reply:
I don't see the issue with existing sent emails. Nothing will have changed about the fact the user has to visit the url `/account/verify/{token}` - it's what happens when they get there that is changing.

Reply:
Can I clarify please - is the suggestion we should change the API route 9move it out of the way) and leave the frontend route the same?

Reply:
Can it not just be GET for the webpage and POST to do the update?

Reply:
@hardillb yes, I think so - as I wrote below in response to Nick, my brain has just kicked in 🙄